src/PIL/ImageFilter.py

```diff
@@ -149,9 +149,10 @@ def filter(self, image):
 
 
 class GaussianBlur(MultibandFilter):
-    """Gaussian blur filter.
+    """Blurs the image with an extended box filter, which approximates a
```

Comment:
"with a sequence of extended box filters"?

Reply:
Done